apps/metastore/src/metastore/views.py

```diff
@@ -256,6 +260,51 @@ def get_table_metadata(request, database, table):
   return JsonResponse(response)
 
 
+def get_table_health_status(table, at_time=None):
+  """Returns table health status."""
```

Comment:
let's describe the main idea:
- we have crontab definition for the expected schedule
- we use fields `A`, `B` to get the last export time